### PR TITLE
Towards #464: Revert grid node size changes

### PR DESCRIPTION
**Pull request by rylin8**
The grid node sizes output from the list widget preview are no longer needed. Previously, the children of lazy grids were a Box() with a size retrieved from layout. The layout retrieval was being called in the DesignFrame() composable before layout was complete, so layout could be null. Now the children of lazy grids use our custom DesignFrameLayout, where layout is retrieved in the layout phase after all the DesignFrame composables have been run, so layout is not null.

### CODE REVIEW

**Review by timothyfroehlich — CHANGES_REQUESTED** [review]
Verify roborazzi failed, so we have a screenshot diff. I'll click the update branch button to rebase this and have the updated comment bot post the difference.

If you're fine with the differences then you'll need to run `./gradlew recordRoborazziDebug` to capture new screenshots and commit them.

File: support-figma/auto-content-preview-widget/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "list-preview-widget",
-  "version": "1.1.0",
+  "version": "1.0.1",
```
Comment (timothyfroehlich) [inline support-figma/auto-content-preview-widget/package.json:3]: Was this version change intentional?

**Review by timothyfroehlich — CHANGES_REQUESTED** [review]

**Comment by rylin8** [comment]
> Verify roborazzi failed, so we have a screenshot diff. I'll click the update branch button to rebase this and have the updated comment bot post the difference.
> 
> If you're fine with the differences then you'll need to run `./gradlew recordRoborazziDebug` to capture new screenshots and commit them.

Hmm I did run that already and the changes are in here I thought.

File: support-figma/auto-content-preview-widget/package.json (hunk 1 above)
Comment (rylin8) [reply support-figma/auto-content-preview-widget/package.json:3]: Yes, I increased changed to 1.1.0 previously and am reverting that change.

**Comment by timothyfroehlich** [comment]
> > Verify roborazzi failed, so we have a screenshot diff. I'll click the update branch button to rebase this and have the updated comment bot post the difference.
> > If you're fine with the differences then you'll need to run `./gradlew recordRoborazziDebug` to capture new screenshots and commit them.
> 
> Hmm I did run that already and the changes are in here I thought.

I see that, and it is the layout one that's failing, it seems:
```
com.android.designcompose.testapp.validation.RenderAllExamples > testRender[TestConfig(fileName=Layout Tests, fileComposable=androidx.compose.runtime.internal.ComposableLambdaImpl@74f6c5d8, fileClass=com.android.designcompose.testapp.validation.examples.LayoutTestsDoc)] FAILED
```

**Comment by rylin8** [comment]
I'll rerun recordRoborazziDebug to get a new screenshot for the layout test. I think I had a local change that I didn't commit which is why it's different. The layout screenshot is going to be changing a lot until the layout changes are done as that's the one I'm constantly editing to test.

File: integration-tests/validation/src/testDebug/roborazzi/RenderAllExamples/Layout-Tests.png
Comment (timothyfroehlich) [inline integration-tests/validation/src/testDebug/roborazzi/RenderAllExamples/Layout-Tests.png:1]: This change is expected, right?

**Review by timothyfroehlich — APPROVED** [review]

File: integration-tests/validation/src/testDebug/roborazzi/RenderAllExamples/Layout-Tests.png
Comment (iamralpht) [reply integration-tests/validation/src/testDebug/roborazzi/RenderAllExamples/Layout-Tests.png:1]: also the dcf file got larger, so there were some changes in the source file...

Comment (rylin8) [reply integration-tests/validation/src/testDebug/roborazzi/RenderAllExamples/Layout-Tests.png:1]: Yes, I am constantly changing LayoutTests so it changes frequently.

**Pull request merged**